src/components/atoms/newsletter/Navigation/styles.jsx

```diff
@@ -0,0 +1,43 @@
+import { styled } from 'styled-components';
+
+export const Wrapper = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+`;
+
+export const PageNavigation = styled.div`
+  display: flex;
+  gap: 10px;
+
+  span {
+    border-radius: 50%;
+    width: 20px;
+    height: 20px;
+    background-color: grey;
+
+    &:hover,
+    &:focus,
+    &:active {
+      cursor: pointer;
+      background-color: #222;
+    }
+  }
+
+  span.active {
+    background-color: #000;
+  }
+`;
+
+export const NavButton = styled.button`
```

Comment:
The size of the navigation arrows and circles should be small for mobile screens.

Reply:
@Lucif3r-in Done